### PR TITLE
Parquet 2 csv

### PR DESCRIPTION
**Pull request by hayssams**
## Summary
We often need a way to export data as csv files once they have been ingested through Comet. 

**PR Type: Feature**

**Status: Ready**

**Breaking change? No**

### Solution
This PR implements a simple parquet to CSV files that may be called with : 
- Input folder only : In this case all domains & schema in this folder are converted to csv
- Input folder & Domain Name only : In this case all dataset  in the specified domain are concerted to csv
- Input folder & Domain Name and Schema Name : In this case its will convert the dataset at this schema location is converted to csv

Command Line arguments also include the option to add a header, to specify a separator, to require a number of output partitions.
### How has this been tested?
Tested by generating a parquet file from a list of case classes and checked that the converted csv is located in the right folder that the the parquet & csv files both contain the same list of records

### CODE REVIEW

File: src/main/scala/com/ebiznext/comet/job/convert/Parquet2CSV.scala [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          .list(config.inputFolder)
+          .flatMap(domainPath => storageHandler.list(domainPath))
+    }
+    val outputPath = config.outputFolder match {
+      case None         => config.inputFolder
+      case Some(folder) => folder
+    }
+    allPaths.flatMap { path: Path =>
+      val successPath = new Path(path, "_SUCCESS")
+      println("par"+config.partitions)
```
Comment (mhdkassir) [inline src/main/scala/com/ebiznext/comet/job/convert/Parquet2CSV.scala]: looks like a debug message, to be deleted ?

File: src/main/scala/com/ebiznext/comet/job/bqload/BigQueryLoadConfig.scala [hunk 2]
```diff
@@ -1,5 +1,9 @@
 package com.ebiznext.comet.job.bqload
 
+import buildinfo.BuildInfo
+import com.ebiznext.comet.job.ingest.IngestConfig
```
Comment (mhdkassir) [inline src/main/scala/com/ebiznext/comet/job/bqload/BigQueryLoadConfig.scala]: unused imports to clean

File: project/Common.scala [hunk 3]
```diff
@@ -27,6 +27,8 @@ import org.scalafmt.sbt.ScalafmtPlugin.autoImport.scalafmtOnCompile
 import sbt.Keys._
 import sbt.{Def, _}
 import sbtassembly.AssemblyKeys._
+import sbtbuildinfo.BuildInfoPlugin.autoImport.{BuildInfoKey, buildInfoKeys, buildInfoPackage}
+import sbtbuildinfo.{BuildInfoKey, BuildInfoPlugin}
```
[the remark this answers is not in the document]
Comment (hayssams) [reply project/Common.scala]: Done

File: src/main/scala/com/ebiznext/comet/job/convert/Parquet2CSVConfig.scala [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+import buildinfo.BuildInfo
+import com.ebiznext.comet.schema.model.WriteMode
+import com.ebiznext.comet.utils.CliConfig
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.SaveMode
+import scopt.OParser
+
+case class Parquet2CSVConfig(
+  val inputFolder: Path = new Path("/"),
```
[the remark this answers is not in the document]
Comment (hayssams) [reply src/main/scala/com/ebiznext/comet/job/convert/Parquet2CSVConfig.scala]: done

**Pull request merged**